burn-import/src/onnx/op_configuration.rs

```diff
@@ -357,3 +392,31 @@ fn padding_config(pads: &[i64]) -> PaddingConfig2d {
         panic!("Padding configuration ({:?}) not supported", pads);
     }
 }
+
+pub fn reshape_config(node: &Node) -> Vec<i64> {
+    let mut allowzero = 0;
+
+    for (key, value) in node.attrs.iter() {
+        match key.as_str() {
+            "allowzero" => attr_value_i64(value, &mut allowzero),
+            _ => {}
+        }
+    }
+
+    // Burn does not support zero size shape
+    if allowzero != 0 {
+        panic!("Zero shape size is not supported");
+    }
```

Comment:
Not sure why allowzero is an int and not a bool. What am I missing here?

Reply:
This is from ONNX. In the spec, they use 0 or 1. I'll update with a documentation link. 

https://onnx.ai/onnx/operators/onnx__Reshape.html#attributes

Reply:
Ho well 😅